handler/src/main/java/io/netty/handler/ssl/OpenSslContext.java

```diff
@@ -501,8 +524,52 @@ public OpenSslEngine remove(long ssl) {
         public void add(OpenSslEngine engine) {
             engines.put(engine.sslPointer(), engine);
         }
+
+        @Override
+        public OpenSslEngine get(long ssl) {
+            return engines.get(ssl);
+        }
     }
 
+    static void setKeyMaterial(long ctx, X509Certificate[] keyCertChain, PrivateKey key, String keyPassword)
+            throws SSLException {
+         /* Load the certificate file and private key. */
+        long keyBio = 0;
```

Comment:
I haven't seen a system where `NULL` is not zero ... but for completeness we should consider getting `NULL` as defined from JNI if we intended to use longs as pointers ... last I checked there wasn't anything in the language that required `NULL` to be `0`.